Sources/AudioKit/Taps/NodeRecorder.swift

```diff
@@ -80,22 +82,27 @@ open class NodeRecorder: NSObject {
     ///   - fileDirectoryPath: Directory to write audio files to
     ///   - bus: Integer index of the bus to use
     ///   - rawDataTapHandler: Raw audio data callback
+    ///   - shouldCleanupRecordings: Determines if recorded files are deleted upon deinit (default = true)
     ///
     public init(node: Node,
                 fileDirectoryURL: URL? = nil,
                 bus: Int = 0,
-                rawDataTapHandler: RawAudioDataHandler? = nil) throws {
+                rawDataTapHandler: RawAudioDataHandler? = nil,
+                shouldCleanupRecordings: Bool = true) throws {
```

Comment:
I think that you should leave the rawDataTapHandler as the last item in the initializer so that it can be a trailing closure.

Reply:
Good call! Just switched the order